cmd/agent/app/builder_test.go

```diff
@@ -235,50 +233,57 @@ func TestCreateCollectorProxy(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		flags := &flag.FlagSet{}
-		grpc.AddFlags(flags)
-		reporter.AddFlags(flags)
+		t.Run("", func(t *testing.T) {
+			flags := &flag.FlagSet{}
+			grpc.AddFlags(flags)
+			reporter.AddFlags(flags)
 
-		command := cobra.Command{}
-		command.PersistentFlags().AddGoFlagSet(flags)
-		v := viper.New()
-		v.BindPFlags(command.PersistentFlags())
+			command := cobra.Command{}
+			command.PersistentFlags().AddGoFlagSet(flags)
+			v := viper.New()
+			v.BindPFlags(command.PersistentFlags())
```

Comment:
Replace above with
```
v, command := config.Viperize(grpc.AddFlags, reporter.AddFlags)
```

Reply:
Should I push this change in the next PR?

Reply:
+1